helm/fiftyone-teams-app/README.md

```diff
@@ -571,17 +575,7 @@ upgrading from FiftyOne Teams version 1.1.0 or later:
 A minimal example `values.yaml` may be found
 [here](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/values.yaml).
 
-1. Edit the `values.yaml` file, setting these required customer provided settings
-    1. Secrets
-        1. `secret.fiftyone.mongodbConnectionString`
-        1. `secret.fiftyone.cookieSecret`
-        1. `secret.fiftyone.encryptionKey`
-        1. `secret.fiftyone.fiftyoneAuthSecret`
-
-        > **Note**: All the secret fields defined in the Helm chart must be set.
-        > If `secret.create=false`, the secret you provide must contain all the fields
-        > in the chart's `secret.fiftyone`.
-    1. `teamsAppSettings.dnsName`
```

Comment:
Might need to rebase to not remove these lines?

Reply:
doesn't look like those lines are in `release/v1.6.0`